packages/ckeditor5-list/src/documentlist/documentlistediting.ts

```diff
@@ -506,12 +511,17 @@ export default class DocumentListEditing extends Plugin {
 		//	                       │  * bar]             │ * bar             │
 		//	                       └─────────────────────┴───────────────────┘
 		//
-		// See https://github.com/ckeditor/ckeditor5/issues/11608.
-		this.listenTo<ModelGetSelectedContentEvent>( model, 'getSelectedContent', ( evt, [ selection ] ) => {
-			const isSingleListItemSelected = isSingleListItem( Array.from( selection.getSelectedBlocks() ) );
+		// See https://github.com/ckeditor/ckeditor5/issues/11608, https://github.com/ckeditor/ckeditor5/issues/14969
+		this.listenTo<ViewDocumentClipboardOutputEvent>( viewDoc, 'clipboardOutput', ( evt, data ) => {
+			if ( data.method != 'dragstart' ) {
+				const selection = model.document.selection;
+				const isSingleListItemSelected = isSingleListItem( Array.from( selection.getSelectedBlocks() ) );
 
-			if ( isSingleListItemSelected ) {
-				model.change( writer => removeListAttributes( Array.from( evt.return!.getChildren() as any ), writer ) );
+				if ( isSingleListItemSelected ) {
+					const content = this.editor.data.toModel( data.content );
```

Comment:
This does not alter the clipboard content. As talked f2f let's introduce the `outputTransformation` event.